src/App/Order/Collection/PdkOrderCollection.php

```diff
@@ -42,9 +42,11 @@ public function generateShipments(): ShipmentCollection
         $newShipments = new ShipmentCollection();
 
         $this->each(function (PdkOrder $order) use ($newShipments) {
-            $newShipment = $order->createShipment();
-            $newShipments->push($newShipment);
-            $order->shipments->push($newShipment);
+            for ($i = 0; $i < $order->deliveryOptions->labelAmount; $i++) {
```

Comment:
Maak je nou ook meerdere shipments aan als er echte multicollo is? Dat lijkt me niet de bedoeling

Reply:
Jaa dat doe ik omdat je later bij encodeSecondaryShipments die shipments weer nodig hebt